core/src/main/java/org/apache/iceberg/jdbc/JdbcUtil.java

```diff
@@ -123,7 +123,7 @@ enum SchemaVersion {
           + JdbcTableOperations.METADATA_LOCATION_PROP
           + " = ?";
   static final String V0_CREATE_CATALOG_SQL =
-      "CREATE TABLE "
+      "CREATE TABLE IF NOT EXISTS "
```

Comment:
Afair, it has been discussed while ago.

As this syntax is not supported by all SQL backends, we control that via `JdbcUtil.INIT_CATALOG_TABLES_PROPERTY`.

You can see the login in the `initializeCatalogTables()` method.